aspnetcore/blazor/host-and-deploy/server.md

```diff
@@ -66,87 +66,52 @@ Recommendations for global deployments to geographical data centers:
+Hosting on Azure App Service requires configuration for WebSockets and session affinity, also called Application Request Routing (ARR) affinity.
 
-:::moniker-end
+See the following resources:
 
-> [!IMPORTANT]
-> When [WebSockets](https://wikipedia.org/wiki/WebSocket) are disabled, Azure App Service simulates a real-time connection using HTTP Long Polling. HTTP Long Polling is noticeably slower than running with WebSockets enabled, which doesn't use polling to simulate a client-server connection. In the event that Long Polling must be used, you may need to configure the maximum poll interval (`MaxPollIntervalInSeconds`), which defines the maximum poll interval allowed for Long Polling connections in [Azure SignalR Service](#azure-signalr-service) if the service ever falls back from WebSockets to Long Polling. If the next poll request does not come in within `MaxPollIntervalInSeconds`, Azure SignalR Service cleans up the client connection. Note that Azure SignalR Service also cleans up connections when cached waiting to write buffer size is greater than 1 MB to ensure service performance. Default value for `MaxPollIntervalInSeconds` is 5 seconds. The setting is limited to 1-300 seconds.
->
-> **We recommend using WebSockets for server-side Blazor apps deployed to Azure App Service.** The [Azure SignalR Service](xref:signalr/scale#azure-signalr-service) uses WebSockets. If the app doesn't use the Azure SignalR Service, see <xref:signalr/publish-to-azure-web-app#configure-the-app-in-azure-app-service>.
->
-> For more information, see:
->
-> * [What is Azure SignalR Service?](/azure/azure-signalr/signalr-overview)
-> * [Performance guide for Azure SignalR Service](/azure/azure-signalr/signalr-concept-performance#performance-factors)
-> * <xref:signalr/publish-to-azure-web-app>
+* [Configure the app in Azure App Service](xref:signalr/publish-to-azure-web-app#configure-the-app-in-azure-app-service)
+* [App Service Plan Limits](xref:signalr/publish-to-azure-web-app#app-service-plan-limits)
 
-The app must support *session affinity*, also called *sticky sessions*, where clients are redirected back to the same server. To support session affinity, an application setting in the Azure portal is automatically configured (**Configuration** > **Application settings**: **Name**: `Azure__SignalR__ServerStickyMode`, **Value**: `Required`). Therefore, you don't need to manually configure the app for session affinity.
+If the app is deployed to a single server to run in a single process, the app doesn't require using the [Azure SignalR Service](#azure-signalr-service).
 
-If you prefer not to use the app setting in Azure (and delete it in the Azure portal), two other approaches for configuring session affinity are (**use either approach, not both**):
+If the app is deployed to a webfarm (multiple servers), the [Azure SignalR Service](#azure-signalr-service) is recommended.
```

Comment:
I don't think this is accurate. If you're Blazor app is hosted on Azure App Service, then you don't really need the Azure SignalR Service

Reply:
> For Blazor apps, we recommend using the Azure SignalR Service to:
>
> * Facilitate connection scale out
> * Handle global distribution

That's why I used the word "recommended." On more than one instance, isn't it recommended? And, doesn't "scale out" mean more than one instance?

The SignalR doc recommends it for scale out ... last line of the section (under the image) at ...

https://learn.microsoft.com/en-us/aspnet/core/signalr/scale?view=aspnetcore-8.0#scale-out

... and the first line of ...

https://learn.microsoft.com/en-us/aspnet/core/signalr/scale?view=aspnetcore-8.0#azure-signalr-service

Reply:
When you publish to Azure App Service with multiple instances you're effectively on a web farm, but I believe you still don't need the Azure SignalR Service.

With the Azure SignalR Service, connection scale out means that it will multiplex multiple connections to reduce the connection load on your servers: https://learn.microsoft.com/azure/azure-signalr/signalr-concept-internals#application-server-connections.

Global distribution means the Azure SignalR Service is available in multiple regions and backed by a fast network.

Reply:
Yes, session affinity and WebSockets are the only requirements from what I'm reading.

Still tho, I was hoping for a firmer recommendation for using the service, which we know offloads those two items from the app. I guess I should just go with your text ...

> For Blazor apps, we recommend using the Azure SignalR Service to:
>
> Facilitate connection scale out
> Handle global distribution

... and the cross-links and let the reader decide from there. I was probably reading too much into the references. They seem to kind'a strongly recommend it for app scale-out (multiple instances/multiple servers), and maybe that's because $$ is at stake.

Reply:
There are other reasons to use the Azure SignalR Service when working directly with SignalR (like using the service as a SignalR backplane), but with Blazor the benefits are more scoped.